### PR TITLE
Fix confusing output when cancelling deployments

### LINKED ISSUE #3545: Cancelling deployment of a running deployment config shows incorrect messages
Have 3 failed deployments (1, 2, and 4), one successful deployment (3), and a running deployment (5)

```
$ oc deploy master --cancel
no active deployments to cancel
no active deployments to cancel
no active deployments to cancel
no active deployments to cancel
cancelled #5
```

Should have only written "Cancelled deployment #5"

```
$ oc get pods
NAME              READY     REASON         RESTARTS   AGE
master-1-deploy   0/1       ExitCode:255   0          41m
master-2-deploy   0/1       ExitCode:255   0          38m
master-3-8jlfb    0/1       Running        12         33m
master-4-deploy   0/1       ExitCode:255   0          26m
master-5-bimfz    0/1       Running        7          1m
master-5-deploy   1/1       Running        0          1m
```

### PR DESCRIPTION
**Pull request by ironcladlou**
Provide more accurate user messaging when cancelling deployments.

Fixes #3545

### CODE REVIEW

**Comment by ironcladlou** [comment]
@kargakis PTAL

File: pkg/cmd/cli/cmd/deploy.go [hunk 1]
```diff
@@ -331,19 +332,20 @@ func (c *cancelDeploymentCommand) cancel(config *deployapi.DeploymentConfig, out
 			_, err := c.client.UpdateDeployment(&deployment)
 			if err == nil {
 				fmt.Fprintf(out, "cancelled #%d\n", config.LatestVersion)
```
Comment (kargakis) [inline pkg/cmd/cli/cmd/deploy.go]: Make this "Cancelled deployment #" as Clayton suggested

**Comment by kargakis** [comment]
nit aside, LGTM

File: pkg/cmd/cli/cmd/deploy.go (hunk 1 above)
Comment (ironcladlou) [inline pkg/cmd/cli/cmd/deploy.go]: Fixed

**Comment by kargakis** [comment]
[merge]

**Pull request merged**